src/tools/rustdoc-js/tester.js

```diff
@@ -190,6 +190,31 @@ function loadThings(thingsToLoad, kindOfLoad, funcToCall, fileContent) {
     return content;
 }
 
+function contentToDiffLine(key, value) {
+    return `"${key}": "${value}",`;
+}
+
+function betterLookingDiff(entry, data) {
+    let output = '{\n';
+    let error = false;
+    for (var i = 0; i < data.length && error === false; ++i) {
```

Comment:
This will stop after the first error, which means you may need to run the test multiple times to see everything that went wrong. Would it be better to show all the errors at once instead?
```suggestion
    for (var i = 0; i < data.length; ++i) {
```

Reply:
Explanation time! In case you have multiple results in your search, it'll go through all of them, then making the diff on all the results. Which is why I wrote `Diff on the first result`. ;)

I'll add a comment in the code explaining that!

Reply:
Hum actually it allowed me to improve the code. So I added the comment explaining a bit more and improved code. That's a double win situation hehe.

Reply:
Yeah, I was wondering about that too.

Reply:
Oops, that comment is out of context; I was responding to jyn514's post, but I hadn't seen Guillaume's comments because GitHub sometimes doesn't reload properly...